Makefile

```diff
@@ -0,0 +1,12 @@
+SHELL := /bin/bash
+
+install:
+	pdm install -dG:all
+
+docs:
+	mkdocs serve --livereload -w srai
+
+test:
+	pytest -n auto
```

Comment:
Are we going to use pytest, or tox?

Reply:
i don't see any advantage as of now for tox. Locally I'm always running it like this, but up for a debate